eventhubskafka-flink-eventhubskafka/create-solution.sh

```diff
@@ -209,8 +209,9 @@ echo
 
 echo "***** [P] Setting up PROCESSING"
 
+    export APPINSIGHTS_NAME=$PREFIX"appmon"
     export LOG_ANALYTICS_WORKSPACE=$PREFIX"mon"
```

Comment:
is there any possibility to have all logs (Flink + AKS) combined in a single place?

Reply:
I guess so. I will look into this.